lib/json/ld/context.ex

```diff
@@ -46,7 +46,47 @@ defmodule JSON.LD.Context do
 
   # 3.2) If context is a string, [it's interpreted as a remote context]
   defp do_update(%JSON.LD.Context{} = active, local, remote, options) when is_binary(local) do
-    # TODO: fetch remote context and call recursively with remote updated
+    # 3.2.1)
+    local = absolute_iri(local, base(active))
+
+    # 3.2.2)
+    remote = case Enum.member?(remote, local) do
+      false -> [local | remote]
```

Comment:
I think the performance benefit of prepending is not worth the effort. We diverge from the spec, have to reverse on its use anyway, but most importantly the `remote` argument comes from `JSON.LD.Context.update` which is part of the public API, so we can not reverse (unless requiring the user to pass them in reversed order in the first place).